### ISSUE TITLE
Unable to wakeup and reboots unexpectedly

### ISSUE THREAD

**Issue by begundal** [LegacyHuawei/local_manifests#6]
Hello,
I'm experiencing the phone unable to wakeup, I must pull of battery to revive it also I'm experiencing unexpectedly reboots.
I've been experiencing these since cm-12.1-20150815-UNOFFICIAL-LegacyHuawei-u8800.zip after I upgraded to cm-12.1-20150922-UNOFFICIAL-LegacyHuawei-u8800.zip and cm-12.1-20150922-UNOFFICIAL-LegacyHuawei-u8800-update1.zip the problem persists. Unfortunately I can not reproduced the occurrences
As for the logs I upload to Google Drive https://goo.gl/X6Y0oV

**Comment by rtammekivi** [comment]
Can you try reverting to CFQ IO scheduler? Seems like it might be the reason why it's happening.

**Comment by eliotcougar** [comment]
Yes, I've had unexpected reboots twice since I flashed 20150922 on my U8860... It's not reproducible and totally random...
How do I change the scheduler since there is no menu for that anymore?..

**Comment by rtammekivi** [comment]
You can do it via a third party app, or it should be available to change in build.prop aswell (sys.io.scheduler or similar)

**Comment by begundal** [comment]
I'm using https://play.google.com/store/apps/details?id=com.grarak.kerneladiutor to change both internal and external storage i/o scheduler from bfq to cfq.
Will keep you posted should the random reboot occur again.

**Comment by begundal** [comment]
random reboot still occur after changing from bfq to cfq
is it possible because my external sdcard faulty
latest dontpanic files https://goo.gl/QuKYOQ

**Comment by eliotcougar** [comment]
I notice that random reboots happen only after I use some heavy apps (i.e. Google Maps, OpenCamera, Google Translate in the Live Translate mode)...
Those apps are very laggy, almost unusable on U8860... The device may work for a long time without a reboot if I don't launch them, but if I do, it reboots later in the day...
